todo.css

```diff
@@ -3,23 +3,23 @@
   box-shadow: 0 1px 3px 0 rgba(0,0,0,.1);
 }
 
- .container {
+.container {
   width:700px;
-  margin-right: auto;
-  margin-left : auto;
+  margin: 0 auto;
 }
 
 .box1 {
-    padding: 0.5em 1em;
-    margin: 2em 0;
-    width: 80%;
-    color: #5d627b;
-    background: white;
-    box-shadow: 0 3px 5px rgba(0, 0, 0, 0.22);
+  padding: 0.5em 1em;
```

Comment:
I personally have a strong opinion against using em and rem unless you're planning to scale your interface somehow.

A long time ago `em` was necessary because the browsers didn't have an actual zoom, it simply changed the font-size. Modern browsers don't do this anymore, they change the pixel size.

You can research more about it but I recommend not using em or rem unless necessary.